commit fa7ad1df1915c8a62f50e3a5b7e10f9c7118cd7f
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Sun Feb 6 11:12:24 2022 -0800

    elf: Replace tst-audit24bmod2.so with tst-audit24bmod2
    
    Replace tst-audit24bmod2.so with tst-audit24bmod2 to silence:
    
    make[2]: Entering directory '/export/gnu/import/git/gitlab/x86-glibc/elf'
    Makefile:2201: warning: overriding recipe for target '/export/build/gnu/tools-build/glibc-gitlab/build-x86_64-linux/elf/tst-audit24bmod2.so'
    ../Makerules:765: warning: ignoring old recipe for target '/export/build/gnu/tools-build/glibc-gitlab/build-x86_64-linux/elf/tst-audit24bmod2.so'

Conflicts:
	elf/Makefile

diff --git a/elf/Makefile b/elf/Makefile
index 4d16ed1637db8582..73d347339762fc9e 100644
--- a/elf/Makefile
+++ b/elf/Makefile
@@ -855,7 +855,7 @@ extra-test-objs += $(addsuffix .os,$(strip $(modules-names)))
 
 # filtmod1.so has a special rule
 modules-names-nobuild := filtmod1 \
-			 tst-audit24bmod1 tst-audit24bmod2.so
+			 tst-audit24bmod1 tst-audit24bmod2
 
 tests += $(tests-static)
 
